sdk/python/MANIFEST.in

```diff
@@ -1 +1 @@
-include kfp/dsl/type_schemas/*.yaml
```

Comment:
These yaml files were not really used since: https://github.com/kubeflow/pipelines/commit/1a357a0ebe3b25ffdc2edfabdb1de1f7861e0901